lib/http.js

```diff
@@ -116,6 +117,18 @@ function get(url, options, cb) {
   return req;
 }
 
+/**
+ * Lazy loads WebSocket, CloseEvent and MessageEvent classes from undici
+ * @returns {object} An object containing WebSocket, CloseEvent, and MessageEvent classes.
+ */
+function lazyWebSocket() {
+  if (!WebSocket) {
+    const { WebSocket, CloseEvent, MessageEvent } = require('internal/deps/undici/undici');
+    return { WebSocket, CloseEvent, MessageEvent };
+  }
+  return { WebSocket, CloseEvent, MessageEvent };
```

Comment:
If three different things are being exported here, why not just export the entire undici`? E.G.
```suggestion
function lazyUndici() {
  undici ??= require('internal/deps/undici/undici');
  return undici;
```

Reply:
This is what I said before.

Reply:
I know, I wanted to reiterate with this function, as the other conversation was marked as resolved.

Reply:
Fixed.